prime-dotnet-webapi/Controllers/ClientLogsController.cs

```diff
@@ -0,0 +1,37 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+using Prime.Services;
+using Prime.Models;
+using Prime.ViewModels;
+
+namespace Prime.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/client-logs")]
+    [ApiController]
+    public class ClientLogsController : PrimeControllerBase
+    {
+        private readonly IClientLogService _logService;
+
+        public ClientLogsController(IClientLogService logService)
+        {
+            _logService = logService;
+        }
+
+        // POST /api/client-logs
+        /// <summary>
+        /// Creates a client log
+        /// </summary>
+        [HttpPost("", Name = nameof(CreateLog))]
+        [AllowAnonymous]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        public async Task<ActionResult> CreateLog(ClientLogViewModel log)
+        {
+            await _logService.CreateLogAsync(log);
```

Comment:
normally I would never advocate for a `try ... catch` to swallow all exceptions, but I think it might be warranted here. If the frontend errors we probably don't want to make the situation worse by accidentally returning another error.

Reply:
So i went the route more that this call on te front-end won't trigger itself, but can double down here too.